packages/api/server.js

```diff
@@ -27,9 +30,11 @@ app.post('/', (req, res) => {
     message: req.body.message ? req.body.message : 'hello world'
   });
 });
+var postBuildHandler = new PostBuildHandler(app, client);
+postBuildHandler.listen();
 
 const port = process.env.PORT ? Number(process.env.PORT) : 3000;
-const server = app.listen(port, () => console.log(`Example app listening at http://localhost:${port}`));
+const server = app.listen(port, 'localhost', () => console.log(`Example app listening at http://localhost:${port}`));
```

Comment:
I would avoid specifying `localhost` here, this will actually cause trouble when we deploy to production.

Is letting it bind to the default port causing issues on your computer?